spec/requests/cloud_volumes_spec.rb

```diff
@@ -192,6 +192,38 @@
       expect(response.parsed_body['data']).to match("form_schema" => {"fields" => []})
       expect(response).to have_http_status(:ok)
     end
+
+    it 'attaches Cloud Volume to an instance' do
+      zone = FactoryBot.create(:zone)
+      ems = FactoryBot.create(:ems_autosde, :zone => zone)
+      vm = FactoryBot.create(:vm_vmware)
+      cloud_volume = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => ems)
+
+      api_basic_authorize(action_identifier(:cloud_volumes, :attach, :resource_actions, :post))
+      stub_supports(cloud_volume.class, :attach)
+
+      payload = {:action => "attach", :resources => {:vm_id => vm.id.to_s}}
+      post(api_cloud_volume_url(nil, cloud_volume), :params => payload)
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'detaches Cloud Volume from an instance' do
+      zone = FactoryBot.create(:zone)
+      ems = FactoryBot.create(:ems_autosde, :zone => zone)
+      hw = FactoryBot.create(:hardware)
+      vm = FactoryBot.create(:vm_vmware, :hardware => hw)
+      cloud_volume = FactoryBot.create(:cloud_volume_autosde, :ext_management_system => ems)
+      disk = FactoryBot.create(:disk, :hardware => hw, :backing => cloud_volume)
```

Comment:
If you don't need to use `disk` later on you just need the record you could do 
```suggestion
      FactoryBot.create(:disk, :hardware => hw, :backing => cloud_volume)
```
And then rubocop won't yell at you